paddle/fluid/inference/api/mkldnn_quantizer_config.cc

```diff
@@ -34,6 +34,9 @@ MkldnnQuantizerConfig::MkldnnQuantizerConfig() {
   rules_["prior_box"]["Image"] = ScaleAlgo::NONE;
   rules_["prior_box"]["Boxes"] = ScaleAlgo::NONE;
   rules_["prior_box"]["Variances"] = ScaleAlgo::NONE;
+
+  rules_["transpose2"]["X"] = ScaleAlgo::KL;
+  rules_["transpose2"]["Out"] = ScaleAlgo::KL;
```

Comment:
Since we overwrite the output scale of the Transpose with input scale, perhaps we should set "Out" scale algorithm to none:
```C++
rules_["transpose2"]["Out"] = ScaleAlgo::NONE;
```
This can also make it less error prone in the future.